docs/health-api-specs/contracts/registries/facility.yml

```
@@ -1,404 +1,640 @@
+      description: Unique id for a tenant.
+      required: true
+      schema:
         type: string
-        example: 'STORAGE_WAREHOUSE'
-      #        enum:
-      #          - STORAGE_WAREHOUSE
-      #          - MEDICAL_FACILITY
-      #          - SEWAGE_TREATMENT_PLANT
-      storageCapacity:
+        format: varchar
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)